src/main/resources/org/jenkinsci/plugins/pipeline/modeldefinition/ModelInterpreter.groovy

```diff
@@ -106,9 +107,10 @@ public class ModelInterpreter implements Serializable {
                                                 firstError = e
                                             }
                                         }
-                                    }
+                                    }.call()
```

Comment:
Honestly? Can't remember exactly. I think it was to deal with making sure we didn't execute at the wrong time...